java/shared/src/test/java/com/squareup/subzero/shared/QrSignerTest.java

```diff
@@ -49,28 +53,26 @@ public void setup() {
             throw new RuntimeException(e);
         }
     }
-    private void writeRandomKey(){
 
-    }
     @Test
-    public void SignTestDeterministic() {
+    public void SignTestDeterministic() throws DestroyFailedException {
         QrSigner signer = new QrSigner();
         byte[] bytes_to_sign = "Subzero".getBytes(StandardCharsets.UTF_8);
         byte[] signature = signer.sign(bytes_to_sign);
         byte[] signature2 = signer.sign(bytes_to_sign);
-        assertThat(signature.equals(signature2));
-        assertThat(signer.verify(signature, bytes_to_sign));
-        assertThat(signer.dumpPublicKey().length == 65);
+        assertArrayEquals(signature, signature2);
+        assertTrue(signer.verify(signature, bytes_to_sign));
+        assertEquals(65, signer.dumpPublicKey().length);
         System.out.println("bytes to sign = " + Hex.toHexString(bytes_to_sign));
         System.out.println("public key = " + Hex.toHexString(signer.dumpPublicKey()));
         System.out.println("Signature = " + Hex.toHexString(signature));
-        assertThat(Hex.toHexString(signature).equals("63ce6fb96b6ee7c0b488304da55bedbd12dd5c19c15cf047df6af56d64f6316062e6b12842919135f498b7d7a963b14c4192ed489ae24f2fe56274408502dd63"));
-        assertThat(Hex.toHexString(signer.dumpPublicKey()).equals("047ce4d183121cb5370619aaa0e0f276ddf972a3ae01fd525633acbdf1ae6076f2f0d251e7a04fabfafdd2bf4b20e9513277a850ab31dd826b474ea5bf73425d36"));
```

Comment:
This was a bug in the old test code. `assertThat` returns an object, but something must be done with it. There should have been an `isTrue()` but there wasn't, so even though the wrong expected values were hard-coded, the test passed.

Reply:
And actually now that I look at it, a bunch of the assertions in this file had the same issue. This test was just completely broken before.